README.md

```diff
@@ -2,35 +2,47 @@
   <br>
   <a href="https://www.asyncapi.org"><img src="https://github.com/asyncapi/parser-nodejs/raw/master/assets/logo.png" alt="AsyncAPI logo" width="200"></a>
   <br>
-  JS Parser
+  JavaScript Parser
 </h5>
-<h4 align="center">Parse and validate AsyncAPI documents</h4>
+<p align="center">
```

Comment:
I liked the previous approach. Short and concise tagline. However, it's true we could improve the description as you did. What about having both? The tagline and the description below.

Reply:
i removed it for 2 reasons, to be consistent with generator and because parser is a name that kind of suggests it is a parser